CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
 Versions
 ========
 
-2017-10-12
-----------
+2017-0X-DEV
```

Comment:
Instead of reverting to `0X-DEV`, you could adjust it to `10-13`, hoping we'll release the new version then.

Reply:
I reverted to 0X-DEV because you told me to. Do I have to stick to 2017-0X-DEV or adjust to 2017-10-13?